front-end-places/Universal.html

```diff
@@ -0,0 +1,4512 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <!--internet explorer-->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Universal</title>
+    <link rel="stylesheet" href="css/vendors-css.min.css">
```

Comment:
replace vendor css paths for: 
    <link rel="stylesheet" type="text/css" href="../app/css/vendors-css.min.css">
    <link rel="stylesheet" type="text/css" href="../app/css/app.min.css">